src/test/java/org/kpmp/datasetSummary/DatasetSummaryTest.java

```diff
@@ -0,0 +1,89 @@
+package org.kpmp.datasetSummary;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.kpmp.DataTypeEnum;
+import org.kpmp.FullDataTypeEnum;
+import org.kpmp.OmicsTypeEnum;
+
+import static org.junit.Assert.assertEquals;
+
+import javax.xml.crypto.Data;
+
+public class DatasetSummaryTest {
+
+    private DatasetSummary datasetSummary;
+
+    @Before
+    public void setUp() {
+    
+        datasetSummary = new DatasetSummary(
+            OmicsTypeEnum.TRANSCRIPTOMICS.getEnum(),
+            FullDataTypeEnum.SINGLE_CELL_FULL.getFull(),
+            DataTypeEnum.SINGLE_CELL.getAbbreviation(),
+            Long.valueOf(2),
```

Comment:
Just a note, you can also do 2l as a short-hand. There is no need to change things here, just pointing it out.